vars/pipelineDeployVegacapsule.groovy

```diff
@@ -261,53 +269,66 @@ void call(Map customConfig = [:]) {
         }
       }
 
-      stage('Sync remote state to local') {
-        options {
-          timeout(5)
-        }
-
-        steps {
-          dir('networks-internal/' + config.networkName + '/vegacapsule') {
-            sh '''
-                mkdir -p "./home"
-                aws s3 sync \
-                --only-show-errors \
-                --no-progress \
-                  "s3://''' + env.S3_BUCKET_NAME + '''/''' + config.networkName + '''" \
-                  "./home/"
-            '''
-            // sh "if [ -d '${env.CONFIG_HOME}/config.hcl' ]; then cat '${env.CONFIG_HOME}/config.hcl'; fi"
+      stage('Prepare deployment') {
+        parallel {
```

Comment:
We do not want to upload those binaries all the time. As this pipeline is generic and We want to keep it optional